lib/install/validate/storage.go

```diff
@@ -175,3 +177,34 @@ func (v *Validator) suggestDatastore(path string, label string, flag string) {
 		}
 	}
 }
+
+func parseScratchSize(input string) int {
```

Comment:
Also I'm doing C-style error reporting here; I can make this function more verbose & use a Go-style error instead if that's more appropriate.